client/my-sites/plugins/plugin-ratings/style.scss

```diff
@@ -9,12 +9,17 @@
 }
 
 .plugin-ratings__rating-stars {
-	font-size: $font-title-medium;
+	display: flex;
+	align-items: center;
 
 	@include breakpoint-deprecated( '<480px' ) {
 		display: inline-block;
 		margin-right: 8px;
 	}
+
+	.rating {
+		margin-right: 5px;
+	}
```

Comment:
Also check if doesn't *break* the version with the feature flag disabled.

Reply:
Done
<img width="795" alt="SS 2021-11-17 at 11 05 40" src="https://user-images.githubusercontent.com/12430020/142170764-41defb78-7b87-478d-b3fc-8a1df5d8ce39.png">